web/static/js/components/css_modules/user_list.css

```diff
@@ -20,7 +20,8 @@
 :global(.basic.segment).index {
   :global(.wrap) {
     max-width: 75rem;
-    max-height: 24rem;
+    max-height: 12rem;
+    min-height: 8rem;
     display: inline-block;
     white-space: initial;
     overflow-x: auto;
```

Comment:
Hmmm... the user list is already `overflow-x: auto;`, so it _is_ scrollable, but I can see how that isn't apparent on the UI, depending on the user's OS settings for showing scroll gutters. 

Does this PR cause largeer user lists to expand on other board layouts like in Idea Submission? The photo in the PR summary only shows the prime directive layout